frontend/site-registry/src/app/features/details/navigation/NavigationPillsConfig.tsx

```diff
@@ -19,16 +20,18 @@ export const navItems: string[] = [
   'Site Disclosure',
 ];
 
-export const navComponents: JSX.Element[] = [
-  <Summary />,
-  <Notations />,
-  <Participants />,
-  <Documents />,
-  <Associate />,
-  <LandUses />,
-  <></>,
-  <Disclosure />,
-];
+export function navComponents(siteId: number): JSX.Element[] {
```

Comment:
I'm not 100% certain that this is the best way to pass the currently viewed site id into my component. Let me know if this is alright or if there is another, better, option that isn't immediately apparent to me.

Reply:
Site ID  can be fetched from the URL.  like this   const { id } = useParams();